CHANGELOG.md

```diff
@@ -1,9 +1,13 @@
 # Changelog
 
-## ✨ yew **0.20.0** *(2022-11-xx)*
+## ✨ yew **0.20.0** *(2022-11-25)*
 
 #### Changelog
 
+- #### 🚨 Breaking changes
```

Comment:
Interesting i never knew we had a breaking changes section before.

Currently we generate the change log using our own generator.
If we add this section back, we need to add it there as well.

Reply:
Though i am not sure f the implications for maintainers, would this mean another label we need to add on PR's?

Reply:
There already is a label, is think it's just not being used in the changelog generator, as far as I see: https://github.com/yewstack/yew/labels/breaking%20change

Reply:
I also think mentioning that `csr` is now a required Cargo feature for old code to continue working is worth mentioning somewhere in the changelog. This doesn't seem directly and explicitly mentioned as far as I can see, and certainly not prominent like in a breaking change section.

Reply:
We never had a section for breaking changes, only migration guide. The breaking change label isn't used in the change log generator

I think this information can go in the migration guide (if not present already)

Reply:
Yes, all the breaking changes are recorded in the migration guide.

I am in favour of having a breaking change section in the changelog.
However, if we were to accept this pull request, we also need to retrospectively pick all breaking changes from the changelog for v0.20.

Maybe it would be better to start tracking breaking changes from the next release?

Reply:
Given this, I would suggest to link the migration guide from the top of the changelog.

I did not know such existed, which was why I was looking at the changelog after my code broke upon updating. I just tried to find the migration guide and had some difficulty with this step. I'm still not sure I was looking at the correct document, but the document I found seemed perhaps a bit incomplete.